### DISCUSSION TITLE
Report generator plugin

### DISCUSSION THREAD

**Discussion by Vladimir-Va** [nocodb/nocodb#373] [Features]
Please think to add:
https://github.com/carboneio/carbone
Fast, Simple and Powerful report generator in any format PDF, DOCX, XLSX, ODT, PPTX, ODS, XML, CSV using your JSON data as input.

**Comment by o1lab** [comment]
Thank you for the link, this will be helpful in exporting the data into different formats.

Notes : 
For PDF - it would need libreoffice server
